core-services/prow/02_config/_plugins.yaml

```diff
@@ -1743,6 +1764,34 @@ plugins:
   - lgtm
   - shrug
   - trigger
+  openshift-kni/cnf-features-deploy:
```

Comment:
Are we *sure* we don't want a top-level configuration for `openshift-kni` org?

Reply:
if that impacts the dev workflow of all the repos in the org, then I'm doubtful we'd want to do something like that.